Dockerfile

```diff
@@ -28,7 +30,7 @@ COPY internal/ internal/
 # was called. For example, if we call make docker-build in a local env which has the Apple Silicon M1 SO
 # the docker BUILDPLATFORM arg will be linux/arm64 when for Apple x86 it will be linux/amd64. Therefore,
 # by leaving it empty we can ensure that the container and binary shipped on it will have the same platform.
-RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o operator cmd/main.go
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=amd64 go build -a -o operator cmd/main.go
```

Comment:
This forces all builds to compile as x86_64. We shouldn't do this on Konflux as it has mechanisms to farm multi-arch builds to VMs running the required architecture natively. In those instances, GOARCH is inherited from the host.

Do we even need to pass in `GOARCH` and `GOOS`?

Reply:
Ohh.. I forgot to revert it. I changed it to test build from mac M1.